woocommerce/woocommerce-ios
# Products Onboarding: Add product_detail_preview_tapped event

Part of https://github.com/woocommerce/woocommerce-ios/issues/8066

## Description

This PR adds a new analytics event for the "Preview" button tap on the product details:

- `*_product_detail_preview_tapped` (Event registration:  1172-gh-tracks-events-registration)

Mobile Tracks spreadsheet updated.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open details for a draft product.
3. Tap the "Preview" button in the navbar. Confirm the event is triggered.

---
